sycl/doc/EnvironmentVariables.md

```diff
@@ -21,6 +21,7 @@ compiler and runtime.
 | `SYCL_CACHE_MIN_DEVICE_IMAGE_SIZE` | Positive integer | Minimum size of device code image in bytes which is reasonable to cache on disk because disk access operation may take more time than do JIT compilation for it. Default value is 0 to cache all images. |
 | `SYCL_CACHE_MAX_DEVICE_IMAGE_SIZE` | Positive integer | Maximum size of device image in bytes which is cached. Too big kernels may overload disk too fast. Default value is 1 GB. |
 | `SYCL_ENABLE_DEFAULT_CONTEXTS` | '1' or '0' | Enable ('1') or disable ('0') creation of default platform contexts in SYCL runtime. The default context for each platform contains all devices in the platform. Refer to [Platform Default Contexts](extensions/PlatformContext/PlatformContext.adoc) extension to learn more. Enabled by default on Linux and disabled on Windows. |
+| `SYCL_USM_HOSTPTR_IMPORT` | Positive integer | Enable by specifying non-zero value. Buffers created with a host pointer will result in host data promotion to USM, improving data transfer performance. To use this feature, also set SYCL_HOST_UNIFIED_MEMORY=1. |
```

Comment:
"Enable by specifying non-zero value." seems to be in conflict with "Positive integer" as non-zero value might be negative.
For my own education: why host data promotion to USM improves data transfer performance? I thought that "buffer" and "USM" should have equivalent data transfer performance if used efficiently. Am I wrong?

Reply:
I mostly agree with you, but saw other instances where one is allowed to set a 0 even though the setting is described as "positive integer". See SYCL_MAX_CACHE_SIZE. I've changed it to "Integer".

Regarding your other question: consider a discrete GPU. The buffer is created as USM device. Data transfer occurs between a normal hostptr and the device USM memory. With this feature, if the hostptr is promoted to USM, both src and dest of data transfer are USM, which is more efficient.